cou/utils/text_styler.py

```diff
@@ -41,33 +39,24 @@ def normal(text: str) -> str:
     return Style.RESET_ALL + text + Style.RESET_ALL
 
 
-def prompt_message(parameter: str, default: Optional[str] = None) -> str:
+def prompt_message(message: str, default_choice: str = "") -> str:
     """Generate eye-catching prompt.
 
-    :param parameter: String to show at the prompt with the user options.
-    :type parameter: str
-    :param default: Default choice if user doesn't provide valid input.
-    :type default: Optional[str]
+    :param message: String to show at the prompt with the user options.
+    :type message: str
+    :param default_choice: Default choice if user doesn't a provide valid input.
+    :type default_choice: str
     :return: Prompt string with the user options.
     :rtype: str
     :raise ValueError: raise ValueError if default choice is invalid
     """
-    continue_option = "y"
-    abort_option = "n"
+    choices = ["y", "n"]
 
-    if not default:  # use all lowercases if no default is passed
-        pass
-    elif default.casefold() == "y":
-        continue_option = "Y"
-    elif default.casefold() == "n":
-        abort_option = "N"
-    else:
-        raise ValueError(f"Invalid default choice: {default}")
+    if default_choice and default_choice.casefold() not in choices:
+        raise ValueError(f"Invalid default choice: {default_choice}")
```

Comment:
not-blocking: I really do not see any benefit using this, if you provide a wrong default value, it will be simply ignored.